imls-directus/README.md

```diff
@@ -0,0 +1,29 @@
+# Directus extensions
+
+Two prototype extensions for wifi sensor work are provided:
+
+- Display: `unix-timestamp`
+  - Converts a unix timestamp to a readable format
+- Panel: `wifi`
+  - Shows daily "wifi minutes served" statistics.
+
+## Building
+
+Run the following to enable automatic reloading of extensions:
```

Comment:
Is this command run from this directory or the root? And do we have to `npm i` or anything first?

Reply:
When I run this locally, it wants a KEY env variable that isn't set yet

Reply:
Oooh yes. You probably want to run `npx directus init` first. That would populate an .env file. 

My .env is stock -- there's nothing I configured in there except to use sqlite3.

Reply:
Good catch, will add to the instructions -- I totally forgot about the env part 😅